test/TEST.groups

```diff
@@ -11,7 +11,7 @@ native_tests = \
     jtreg/generator/test8249757/LibTest8249757Test.java \
     jtreg/generator/test8252016/Test8252016.java \
     jtreg/generator/test8252121/Test8252121.java \
-    jtreg/generator/test8253102/LibTest8253102Test.java \
+    jtreg/generator/reinterpret/TestReinterpret.java \
```

Comment:
uhmmm - I did not realized we had to update this manually (for the other PRs...)

Reply:
Intellij saved me here and caught it here too.